### DISCUSSION TITLE
Real-time data streaming to and from crazyflie via crazyradio

### DISCUSSION THREAD

**Discussion by SkywalkerJLi** [bitcraze/discussions#1003] [Q&A]
I am working on a project that requires streaming real-time data from a base computer to the crazyflie via the crazyradio, so I would like to know if any existing examples can do this. 

I know we can use logging variables to stream data from the crazyflie to the computer, and I was thinking about using the set parameter function to send data to the crazyflie from the computer.

Is this the best way of streaming data or is there a more efficient manner? Thanks!

**Comment by krichardsson** [comment]
There was a related discussion earlier that might be of interest #184 
A good starting point for creating your own packet is the [App channel example](https://github.com/bitcraze/crazyflie-firmware/tree/master/examples/app_appchannel_test)
